bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -227,5 +248,8 @@ spec:
       - name: host-var-lib-cni-networks-openshift-sdn
         hostPath:
           path: /var/lib/cni/networks/openshift-sdn
+      - name: metrics-certs
+        secret:
+          secretName: router-metrics-certs-default
```

Comment:
this should presumably be `sdn-metrics-certs-default`?

Reply:
Oh, it should be tls-metrics-sdn. I copied some bits from the ingress operator in an attempt to make things consistent.